cubrid/CMakeLists.txt

```diff
@@ -286,7 +331,6 @@ install(FILES
   RENAME dbi.h
   )
 
-
 # install pdb files for debugging on windows
 if(WIN32)
   install(DIRECTORY
```

Comment:
If you want all source files (including headers) to be visible in Visual Studio solution/project, grouped eventually:
```
#Visual Studio file filters 
source_group("" FILES ${SRC}) 
source_group("ref" FILES ${SRC_REF})
```

Reply:
For now I was just looking for a way to include the header files in the project (for search/refactoring), I didn't care for how they're organized.

However source_group is definitely something we should consider in the future to improve visualizing our projects in IDE's.

I did remove the if (WIN32) condition for including new headers in their projects.